server/common/network/management/whitelist/post.ts

```diff
@@ -0,0 +1,41 @@
+import {NextApiRequest, NextApiResponse} from "next";
+import {Op} from "sequelize";
+import {isAddress} from "web3-utils";
+
+import Database from "db/models";
+
+import {resJsonMessage} from "../../../../../helpers/res-json-message";
+import {ErrorMessages} from "../../../../errors/error-messages";
+
+export default async function (req: NextApiRequest, res: NextApiResponse) {
+  if (!req.query?.address || req.query?.networkId || (req.query?.address && !isAddress(req.query.address as string)))
+    return resJsonMessage("invalid payload", res, 400);
+
+  const result = await Database.network.find({
+    attributes: ["allow_list"],
+    where: {
+      id: req.query.networkId,
+      allow_list: {
+        [Op.not]: {
+          [Op.contains]: [req.query.address]
+        }
+      }
+    }
+  });
+
+  if (!result)
+    return resJsonMessage(ErrorMessages.NoNetworkFoundOrUserAllowed, res, 400);
+
+
+  const [, updatedAllowList] = await Database.network.update({
```

Comment:
here we can reuse the model returned by result.
Example: 
`
result.allow_list = [...result.allow_list, req.query.address]
await result.save()

return res.status(200).json(result.allow_list);

`